src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1235,37 +1242,20 @@ void ImportLoadIndirect(int token)
 
         void ImportLoadIndirect(TypeDesc type)
         {
-            var value = Pop();
+            ClearPendingPrefix(Prefix.Unaligned);
+            ClearPendingPrefix(Prefix.Volatile);
 
-            CheckIsByRef(value);
+            var address = Pop();
 
+            CheckIsByRef(address);
             if (type != null)
-            {
-                CheckIsAssignablePointer(value.Type, type);
-            }
+              CheckIsAssignablePointer(address.Type, type);
```

Comment:
Nit: Indentation. Also, it  may look better to have `{ }` around it since the else part has it.